<?xml version="1.0" ?>

<launch>
   
   <param name="use_sim_time" value="false"/>
   <arg name="table_id" default="table0"/>

   <!-- BEGIN ROBOT "robot_id" --> 
  <group ns="$(arg table_id)">
    	 <node name="table_agent" pkg="tb_waiter" type="AssignerAgent" output="screen">
    	    <param name="assigner_name"  value="$(arg table_id)"/>
    	 </node>
   
  </group>
    
</launch>




